src/app/core/services/offline-map/offline-map.service.ts

```diff
@@ -512,7 +513,7 @@ export class OfflineMapService {
         encoding: Encoding.ASCII
       });
       const fileStat = await Filesystem.stat({ path });
-      return { size: +fileContent, downloadComplete: fileStat.mtime / 1000 };
+      return { size: +fileContent.data, downloadComplete: fileStat.mtime / 1000 };
```

Comment:
Hvorfor brukes `.data` her? Eventuelt hvorfor brukte vi det ikke tidligere?

Reply:
Aha, nå husker jeg det. Det var feil å ikke ha med .data. Som jeg skriver i PR'en:
"Fikk også retta en feil i OfflineMapService som gjorde at vi ikke visste hvor mye plass en nedlasta pakke tok."
Tror ikke den returnerte noe data i size-feltet før jeg fiksa det. Her er det Filesystem..readFile() returnerer:
```
export interface ReadFileResult {
    /**
     * The string representation of the data contained in the file
     *
     * @since 1.0.0
     */
    data: string;
}
```

Reply:
Hmm, jeg får størrelse på nedlasta kartpakker hos meg?